src/helpers/arraymethodsui.js

```diff
@@ -0,0 +1,10 @@
+// Helaas krijg ik deze functie niet werkend
+export function televisionBrandList(inventory) {
+    // return (
+    //     <ul>
+    //         {inventory.map((product) => {
+    //             return <li key={product.type}>{product.brand}</li>
+    //         })}
+    //     </ul>
+    // )
+}
```

Comment:
Wat je hier doet is eigenlijk een React component maken. De manier zoals je die in je App.jsx importeert is niet de syntax voor React componenten. Zie andere comment